checker/src/org/checkerframework/checker/units/UnitsAnnotatedTypeLoader.java

```diff
@@ -0,0 +1,44 @@
+package org.checkerframework.checker.units;
+
+import org.checkerframework.checker.units.qual.UnitsMultiple;
+import org.checkerframework.common.basetype.BaseTypeChecker;
+import org.checkerframework.framework.type.AnnotatedTypeLoader;
+import org.checkerframework.framework.util.AnnotationBuilder;
+
+import java.lang.annotation.Annotation;
+
+import javax.lang.model.element.AnnotationMirror;
+
+public class UnitsAnnotatedTypeLoader extends AnnotatedTypeLoader {
+
+    public UnitsAnnotatedTypeLoader(BaseTypeChecker checker) {
+        super(checker);
+    }
+
+    /**
+     * Custom filter for units annotations:
+     *
+     * This filter will ignore (by returning false) any units annotation which is an alias of another
+     * base unit annotation (identified via {@link UnitsMultiple} meta-annotation).
```

Comment:
I don't understand this logic: the qualifier is still a supported annotation class, even if it is an alias for a base unit.
If the loader doesn't load this class, can it still be written in the source code?